lib/Container.js

```diff
@@ -138,9 +159,6 @@ Container.prototype.create = Promise.promisify(function(done) {
       `PWD=${SRC_DIR}`,
     ],
   }
-  if (this.build.ref) {
-    createOptions.Env.push(`COMMIT_REF=${this.build.ref}`);
```

Comment:
Yeah, it's already defined on line [#134](https://github.com/ProboCI/probo/pull/8/files#diff-599aa5690d4bc1ef4f297bfe52690761L134).